calvalus-processing/src/main/java/com/bc/calvalus/processing/l2/CombineFileInputFormat.java

```diff
@@ -0,0 +1,83 @@
+package com.bc.calvalus.processing.l2;
+
+import com.bc.calvalus.JobClientsMap;
+import com.bc.calvalus.commons.CalvalusLogger;
+import com.bc.calvalus.commons.InputPathResolver;
+import com.bc.calvalus.inventory.hadoop.FileSystemPathIterator;
+import com.bc.calvalus.inventory.hadoop.HdfsFileSystemService;
+import com.bc.calvalus.processing.JobConfigNames;
+import com.bc.calvalus.processing.geodb.GeodbScanMapper;
+import com.bc.calvalus.processing.hadoop.NoRecordReader;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.LocatedFileStatus;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.fs.RemoteIterator;
+import org.apache.hadoop.mapred.JobConf;
+import org.apache.hadoop.mapreduce.InputFormat;
+import org.apache.hadoop.mapreduce.InputSplit;
+import org.apache.hadoop.mapreduce.JobContext;
+import org.apache.hadoop.mapreduce.RecordReader;
+import org.apache.hadoop.mapreduce.TaskAttemptContext;
+import org.apache.hadoop.mapreduce.lib.input.CombineFileSplit;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * @author thomas
+ */
+public class CombineFileInputFormat extends InputFormat {
```

Comment:
Is there a relation to org.apache.hadoop.mapreduce.lib.input.CombineFileInputFormat ?